sonyc_kalman/kvae/datasets/box.py

```diff
@@ -0,0 +1,127 @@
+import pygame
```

Comment:
Do we need this file? It seems to be all 2d physics functions, I think we can remove this file?